### ISSUE TITLE
Question about lambda

### ISSUE THREAD

**Issue by uigeunahn** [wustl-cig/DOLCE#10]
Hi @JiamingLiu-Jeremy 
What value of lambda did you use for loss function?
Can not find it in the paper.
![image](https://github.com/user-attachments/assets/f4cdd61f-a846-4237-b5d7-004ac2c937dd)
Thank you!

**Comment by JiamingLiu-Jeremy** [comment]
Hi, the lambda value is used only during inference. It is a hyperparameter controlling the strength of the unconditionally trained DDPM (We train a single diffusion model by randomly setting its conditional inputs to zero. As a result, the model, when given zero inputs, acts as an unconditional generative model.).
